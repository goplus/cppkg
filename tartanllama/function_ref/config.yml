name: tl-function-ref
versions:
  1.0.0:
    folder: all
template:
  from: 1.0.0
  folder: all
  tag: v*
