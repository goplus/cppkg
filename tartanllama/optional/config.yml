versions:
  1.0.0:
    folder: all
  1.1.0:
    folder: all
template:
  folder: all
  url: https://github.com/TartanLlama/optional/archive/v${version}.zip
