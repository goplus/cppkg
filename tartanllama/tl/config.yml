name: tl
versions:
  1.0.0:
    folder: all
template:
  from: 1.0.0
  folder: all
  url: https://github.com/TartanLlama/tl/archive/v${version}.tar.gz
