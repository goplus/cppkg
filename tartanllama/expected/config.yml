name: tl-expected
versions:
  1.0.0:
    folder: all
  1.1.0:
    folder: all
  "20190710":
    folder: all
template:
  from: 1.1.0
  folder: all
  tag: v*
